model/work.py

```diff
@@ -1567,6 +1567,7 @@ def query_to_json_array(query):
         # search document.
         search_data = select(
             [works_alias.c.work_id.label("_id"),
+             works_alias.c.work_id.label("work_id"),
```

Comment:
https://www.elastic.co/guide/en/elasticsearch/reference/current/search-request-search-after.html recommends making a copy of the data in `_id` rather than using `_id` as the tiebreaker field.

Reply:
I don't think this per se requires a new migration. The feature branch creates a new version of the index, which should automatically trigger a full index rebuild. All the changes I've been making to this method (with more to come) will take effect then.

We probably need some kind of migration to mark the dropping of ES1 support though.